content/zh/docs/installing-on-kubernetes/faq/configure-a-booster-for-installation/为安装配置加速器.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "为安装配置加速器"
+keywords: 'KubeSphere, 加速器, 安装, FAQ'
+description: '如何为安装配置加速器'
+linkTitle: "为安装配置加速器"
+weight: 3610
+---
+
+如果您无法从 `dockerhub.io` 下载镜像，强烈建议您预先配置仓库的镜像地址（即加速器）以加快下载速度。您可以参考[ Docker 官方文档](https://docs.docker.com/registry/recipes/mirror/#configure-the-docker-daemon)，或执行以下步骤。
+
+## 获取加速器地址
+
+您需要获取仓库的一个镜像地址以配置加速器。 以下示例介绍如何从阿里云获取加速器地址。
+
+1. 登录阿里云控制台，在搜索栏中输入“容器镜像服务”，点击搜索结果中的**容器镜像服务**。
+
+   ![container-registry.png](/images/docs/zh-cn/installing-on-kubernetes/faq/configure-a-booster-for-installation/container-registry.PNG)
+
+2. 点击**镜像加速器**。
+
+   ![image-booster](/images/docs/zh-cn/installing-on-kubernetes/faq/configure-a-booster-for-installation/image-booster.PNG)
+
+3. 在如图所示的位置获取**加速器地址**，并按照页面上提供的阿里云官方操作文档配置加速器。
+
+   ![booster-url](/images/docs/zh-cn/installing-on-kubernetes/faq/configure-a-booster-for-installation/booster-url.PNG)
+
+## 配置仓库镜像地址
+
+您可以直接配置 Docker 守护程序，也可以使用 KubeKey 进行配置。
+
+### 配置 Docker 守护程序
+
+{{< notice note >}}
+
+采用此方法，您需要预先安装 Docker。
+
+{{</ notice >}} 
+
+1. 执行如下命令：
+
+   ```bash
+   sudo mkdir -p /etc/docker
+   ```
+
+   ```bash
+   sudo vi /etc/docker/daemon.json
+   ```
+
+2. 在文件中添加 `registry-mirrors` 键值对。
+
+   ```json
+   {
+     "registry-mirrors": ["https://<my-docker-mirror-host>"]
+   }
+   ```
+
+   {{< notice note >}}
+
+    请将命令中的地址替换成您实际的加速器地址。
```

Comment:
Delete all the space before this sentence. There is sth wrong with the format.

![image](https://user-images.githubusercontent.com/65327072/102745545-34ce7380-4397-11eb-928a-c34b092a6324.png)